crates/indexer/src/models/token_models/collection_datas.rs

```diff
@@ -114,10 +126,39 @@ impl CollectionData {
                     description_mutable: collection_data.mutability_config.description,
                     last_transaction_version: txn_version,
                     inserted_at: chrono::Utc::now().naive_utc(),
+                    table_handle,
                 },
             )))
         } else {
             Ok(None)
         }
     }
+
+    pub fn get_collection_creator(
+        conn: &mut PgPoolConnection,
+        table_handle: &str,
+    ) -> anyhow::Result<String> {
+        let mut retried = 0;
+        while retried < QUERY_RETRIES {
+            retried += 1;
+            match CurrentCollectionData::get_by_table_handle(conn, table_handle) {
+                Ok(current_collection_data) => return Ok(current_collection_data.creator_address),
+                Err(_) => {
+                    std::thread::sleep(std::time::Duration::from_millis(QUERY_RETRY_DELAY_MS));
+                }
+            }
```

Comment:
This is the same issue as in Tokens right?

Reply:
I didn't rebase lmao.